components/Tabs/Tabs.js

```diff
@@ -17,46 +20,29 @@ class TabLink {
 
   };
 
-  select() {
-    // Get all of the elements with the tabs-link class
-    // const links;
-
-    // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
-
-    // Add a class named "tabs-link-selected" to this link
-    // this.element;
-    
-    // Call the select method on the item associated with this link
-
+  onLinkSelect() {
+    this.tabItem.select()
```

Comment:
Should have some code to add + remove the `tabs-link-selected` class from the relevant `TabLink` elements as was described on lines 24 & 27.